datasets/case01/queries/9001_quoteTabInBlob.sql

```diff
@@ -0,0 +1,7 @@
+-- Test case-insensitive names for
+-- Qserv spatial restrictor 
+-- inspired by 1004_varObjects.sql
```

Comment:
I think comment is a copy&paste from other file, maybe update it to reflect what this test is really for?